[2023-08-30T08:25:20.290+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_sale_data.insert_result manual__2023-08-30T08:25:14.845848+00:00 [queued]>
[2023-08-30T08:25:20.302+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_sale_data.insert_result manual__2023-08-30T08:25:14.845848+00:00 [queued]>
[2023-08-30T08:25:20.302+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-30T08:25:20.321+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): insert_result> on 2023-08-30 08:25:14.845848+00:00
[2023-08-30T08:25:20.378+0000] {standard_task_runner.py:57} INFO - Started process 469 to run task
[2023-08-30T08:25:20.381+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'process_sale_data', 'insert_result', 'manual__2023-08-30T08:25:14.845848+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpzel4xmn4']
[2023-08-30T08:25:20.384+0000] {standard_task_runner.py:85} INFO - Job 101: Subtask insert_result
[2023-08-30T08:25:20.426+0000] {task_command.py:410} INFO - Running <TaskInstance: process_sale_data.insert_result manual__2023-08-30T08:25:14.845848+00:00 [running]> on host 888526fa4a36
[2023-08-30T08:25:20.490+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_sale_data' AIRFLOW_CTX_TASK_ID='insert_result' AIRFLOW_CTX_EXECUTION_DATE='2023-08-30T08:25:14.845848+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-30T08:25:14.845848+00:00'
[2023-08-30T08:25:20.547+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-08-30T08:25:20.713+0000] {postgres.py:159} INFO - Running copy expert: COPY consumption_alcoholic_20230830 FROM stdin WITH CSV HEADER DELIMITER AS ',';, filename: /opt/***/dags/consumption_alcoholic_20230830.csv
[2023-08-30T08:25:20.721+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-08-30T08:25:20.735+0000] {postgres.py:159} INFO - Running copy expert: COPY consumption_cereals_bakery_20230830FROM stdin WITH CSV HEADER DELIMITER AS ',';, filename: /opt/***/dags/consumption_cereals_bakery_20230830.csv
[2023-08-30T08:25:20.743+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-08-30T08:25:20.747+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/py_callables.py", line 63, in insert_result
    ,cereals_bakery_file_name)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/hooks/postgres.py", line 167, in copy_expert
    cur.copy_expert(sql, file)
psycopg2.errors.SyntaxError: syntax error at or near "stdin"
LINE 1: COPY consumption_cereals_bakery_20230830FROM stdin WITH CSV ...
                                                     ^

[2023-08-30T08:25:20.758+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=process_sale_data, task_id=insert_result, execution_date=20230830T082514, start_date=20230830T082520, end_date=20230830T082520
[2023-08-30T08:25:20.767+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 101 for task insert_result (syntax error at or near "stdin"
LINE 1: COPY consumption_cereals_bakery_20230830FROM stdin WITH CSV ...
                                                     ^
; 469)
[2023-08-30T08:25:20.795+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-08-30T08:25:20.817+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
